cob_omni_drive_controller/src/odom_plugin.cpp

```diff
@@ -157,7 +160,12 @@ class OdometryController: public GeomController<hardware_interface::JointStateIn
                 odom_tf_.transform.translation.x = odom_.pose.pose.position.x;
                 odom_tf_.transform.translation.y = odom_.pose.pose.position.y;
                 odom_tf_.transform.rotation = odom_.pose.pose.orientation;
-
+                if (invert_odom_){
+                    tf::Transform transform;
+                    tf::transformMsgToTF(odom_tf_.transform, transform);
+                    geometry_msgs::Transform inverted_transform_msg;
+                    tf::transformTFToMsg(transform.inverse(), odom_tf_.transform);
```

Comment:
you invert the matrix, but I don't see where the frame_id and child_frame_id are swapped.....

Reply:
I thought it would lead to less confusion when you can still specify the proper `frame_id` and `child_frame_id` - `invert_odom` is just supposed to invert the odometry information (`base` moves in respect to `odom` vs. `odom` needs to be adjusted in respect to the `base` movement)